### ISSUE TITLE
TASK 8

### ISSUE THREAD

**Issue by MScientistCM** [MScientistCM/March2019ToV6Conversion#8]
m. Change the club competition history years: Change years --> (Start year - 2001) Forward --> Go. [History Project v2.42]
Change the nation competition history years: Change years --> (Start year - 2000) Forward --> Go. [History Project v2.42]

**Comment by jacobclear** [comment]
Taken

**Comment by Dermotron** [comment]
> Taken

How, you don't have the necessary History files?

**Comment by Dermotron** [comment]
Done. Sent via pm on the forum

**Comment by jacobclear** [comment]
If you actually checked the forum, you'd know I'm trying to do it all by myself to prove I can actually do this stuff, so I'd like a copy of these changes as well.

**Comment by Dermotron** [comment]
I've seen the forum but you are talking about manually creating 10 years work of histories manually. It makes no sense whatsoever. The Argentina Copa histories do need to be manually create though (task 9)

**Comment by jacobclear** [comment]
And? I can do that. 9 total competitions is nothing

**Comment by Dermotron** [comment]
Where are you getting 9 competitions from? 

There are over 10k lines in the club history file. Why are you planning on re-creating them all manually? Such a waste of time

**Comment by jacobclear** [comment]
I never said I was going to manually change all of it. I'm just doing what that checklist is telling me to do

**Comment by Dermotron** [comment]
> I never said I was going to manually change all of it. I'm just doing what that checklist is telling me to do

How?

**Comment by jacobclear** [comment]
last time I checked you can change it with the official editor.

**Comment by jacobclear** [comment]
But hey, if you don't want me to do it, just say; I wouldn't be shocked.

**Comment by Dermotron** [comment]
> But hey, if you don't want me to do it, just say; I wouldn't be shocked.

I've already mentioned before a lack of attention to detail. It's been marked done in post #4 

And after a load of going round in circles the answer is in the instruction

**Comment by jacobclear** [comment]
Fine. You clearly do not want my help at all, and whilst I am disappointed I'll just have to keep going without a reward of creating something for people to enjoy. I would have loved to have been able to wrap my head around this but nope, someone has to steal the spotlight. Best I go and disappoint GFRay, again.

**Comment by Dermotron** [comment]
1. I've pointed out what's done in this thread > you've ignored
2. I've pointed out what's needs to be done in this thread > you've ignored
3. You are resorting to slandering cos you are not reading something (you didn't edit the Ronaldo bit in time)
4. Why the constant need for sympathy. You are an adult. Behave like one for an hour.
5. Read point 2 again.
6. Read point 5 and then point 2.
7. Enough with the "poor me" attitude. You are putting it on so much you simply aren't even reading what's in front of you.

**Comment by jacobclear** [comment]
I posted in the thread that I wanted to do this myself > you've ignored.
I resort to slandering because others have done the same, and I have read everything that has been posted.
Fine, I'll behave.
Poor me attitude non-existent - I only wanted something to look back on and go, "I did that."
I've payed waaay to much attention to this and have had my time (that I typically use to Skype some buddies and develop my own little project. 

I'm sorry for what I said, but my mind just flipped because I recalled I was going to do some programming today but I also wanted to do this. Now I don't know what to do because I don't want to make errors and make a fool of myself.

I just want something to develop short term that can make me enjoy the game a touch more, is that too much to ask?

GIve me another chance, I'm no Declan Rice.
